### ISSUE TITLE
Krea2模型训练报错

### ISSUE THREAD

**Issue by xihuanbodajingshen** [WalkingMeatAxolotl/AnimaLoraStudio#430]
尝试最新Krea2模型训练模式，训练日志报错：
2026-07-17 07:46:12,603 - INFO - 加载配置文件: /home/user/app/studio_data/projects/1-bismarck/versions/Krea2_v1/config.yaml
model_family='krea2' 不支持这些已启用字段: ['shuffle_caption']，
但是我完全没有找到这个参数，完整参数如下：
model_family: krea2
transformer_path: /mnt/workspace/studio_data/models/diffusion_models/krea2-raw-bf16.safetensors
vae_path: /mnt/workspace/studio_data/models/vae/qwen_image_vae.safetensors
text_encoder_path: /mnt/workspace/studio_data/models/text_encoders/Qwen_Qwen3-VL-4B-Instruct
text_encoder_cache: true
data_dir: /home/user/app/studio_data/projects/1-bismarck/versions/Krea2_v1/train
resolution:
- 1024
aspect_ratio_limit: 2
reg_data_dir: null
reg_caption: null
reg_weight: 1
flip_augment: false
prefer_json: false
caption_comfy_encoding: true
cache_latents: true
vae_cache_batch_size: 0
cache_encode_tiled: false
lora_type: lokr
lora_rank: 16
lora_alpha: 16
lokr_factor: 8
lora_dora: false
lora_rs: false
lora_dropout: 0
lora_rank_dropout: 0
lora_module_dropout: 0
lora_reg_dims: null
epochs: 10
max_steps: 0
batch_size: 4
grad_checkpoint: false
grad_accum: 1
learning_rate: 1
lr_scheduler: none
optimizer_type: prodigy_plus_schedulefree
ppsf_d_coef: 1.2
ppsf_prodigy_steps: 0
ppsf_beta1: 0.9
ppsf_beta2: 0.99
ppsf_split_groups: true
ppsf_split_groups_mean: false
ppsf_use_speed: false
ppsf_fused_back_pass: false
ppsf_use_stableadamw: true
weight_decay: 0
kv_trim: false
vae_tiling: auto
noise_enhancement_type: none
timestep_sampling: krea2_shift
timestep_shift: 3
timestep_mix_low_prob: 0
timestep_schedule_shift: 1
timestep_shift_resolution_aware: false
infonoise_enabled: false
loss_type: huber
huber_c: 0.15
loss_weighting: detail_inv_t
masked_loss: false
weight_cap_ratio: 0
detail_inv_t_min: 1
detail_inv_t_max: 5
grad_clip_max_norm: 1
mixed_precision: bf16
attention_backend: flash_attn
num_workers: 0
output_dir: /home/user/app/studio_data/projects/1-bismarck/versions/Krea2_v1/output
output_name: bismarck_Krea2_v1
save_every_epochs: 1
save_every_steps: 0
save_state_every_epochs: 0
save_state_every_steps: 0
seed: 42
resume_lora: null
resume_state: null
sample_every: 2
sample_steps: 0
sample_infer_steps: 28
sample_cfg_scale: 4.5
sample_sampler_name: euler
sample_scheduler: krea2_shift
sample_width: 832
sample_height: 1216
sample_seed: 0
sample_negative_prompt: ''
sample_prompt: newest, safe, 1girl, masterpiece, best quality,Bismarck-sh, 1girl, solo, breasts, green eyes, tail, smile, full body, white hair, navel, white background, looking at viewer, simple background, short hair, grin, animal ears, hair between eyes, boots, thighhighs, cat tail, medium breasts, cat ears, standing, black footwear, underboob, scarf, bare shoulders, ass visible through thighs, gauntlets, black thighhighs, tsurime, black cape, thigh boots, v-shaped eyebrows, revealing clothes, legs apart, bikini, black bikini, stomach, swimsuit, black scarf, underwear, slit pupils, claws, tachi-e, cape, torn clothes, pale skin, clawed gauntlets, panties, claw (weapon)
sample_prompts: []
eval_validation_enabled: false

补充使用版本是dev @ 1375ec44 · 2026-07-16 22:57

**Comment by WalkingMeatAxolotl** [comment]
k2 还没有完全弄好。还在修改测试。这个问题是前端训练配置中隐藏的字段，在项目临时保存的 yaml 里面使用了默认值，可以快速给 yaml 文件里面的字段改成 false，我还在重构这一块的代码捏

**Comment by WalkingMeatAxolotl** [comment]
dev 上增加修复了

**Comment by WalkingMeatAxolotl** [comment]
0.20.0 正式版修复了
